src/Tribe/Template/Month.php

```diff
@@ -495,10 +495,13 @@ protected function get_title( $original_title, $sep = null ) {
 		private static function view_more_link( $date ) {
 			$day_link       = tribe_get_day_link( $date );
 			$tribe_bar_args = self::get_tribe_bar_args();
+
 			if ( ! empty( $tribe_bar_args ) ) {
+				if ( isset( $tribe_bar_args['tribe_event_display'] ) ) {
```

Comment:
No need to check if the variable is set - just unset it. It won't throw an error if it's not set already. 😉